### ISSUE TITLE
how to generate RGB detection result on test set

### ISSUE THREAD

**Issue by sus17** [zhixinwang/frustum-convnet#43]
Hi,

I know that frustum convnet relies highly on the quality of 2D RGB detection result. In both this repo and f-pointnet, only RGB detection results on train set and val set are given, without testset-related result. Could you please share your 2d detection result on test set you use when report testing AP? And is possible, could you describe the models or methods you use, to generate 2d detection result in KITTI dataset?

Thanks a lot! Looking forward to your early reply.

**Comment by zhixinwang** [comment]
Refer to #10. The methods of 2D detectors are  described clearly in our paper. 
![image](https://user-images.githubusercontent.com/14836781/98466723-4eaf5f00-220c-11eb-9181-dedaf8ceb04e.png)

**Comment by zhixinwang** [comment]
@sus17  The 2D RGB detection results  we use on the KITTI test set can be downloaded from [HERE](https://drive.google.com/file/d/1wQ9WEoG87d_qEY2jgktAYuMX82zMxEk8/view?usp=sharing). If you use them, also consider  to cite following Papers. Thanks.
[1] J. Ren, X. Chen, J. Liu, W. Sun, J. Pang, Q. Yan, Y.-W. Tai, and L. Xu, “Accurate single stage detector using recurrent rolling convolution,” in Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition, 2017, pp. 5420–5428
[2] Z. Cai, Q. Fan, R. S. Feris, and N. Vasconcelos, “A unified multiscale deep convolutional neural network for fast object detection,” in European conference on computer vision. Springer, 2016, pp. 354–370
